code/client/clrcore/External/Blip.cs

```diff
@@ -155,11 +155,11 @@ public enum BlipSprite
 		ArmWrestling,
 		AmmuNationShootingRange = 313,
 		RaceAir,
-		RaceCar,
+		RaceLand,
```

Comment:
This'd break source compatibility with old scripts. Adding a duplicate entry would work fine, instead (i.e. `RaceLand = 315, RaceCar = 315,`).

Reply:
i see 🤔